docs/source/technical_tutorials/search/maap_stac_r.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>I suggest eliminating use of <code>tryCatch</code> here because it is inconsistent with how your handling search requests everywhere else in this notebook.</p><p> </p><p>Further, you have 2 checks on the length of <code>items$features</code>, so perhaps the second block is unnecessary?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/464/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>